src/runtime_src/core/tools/common/XBHelpMenus.cpp

```diff
@@ -711,7 +711,7 @@ XBUtilities::produce_reports( xrt_core::device_collection devices,
         else {
           platform = xrt_core::device_query<xrt_core::query::rom_vbnv>(device);
         }
-      } catch(...) {
+      } catch(const xrt_core::query::exception&) {

```

Comment:
Following a similar use case as mentioned above, have the catch body set the platform name to "<not defined">.

If this is performed, then set the initial value of platform to an empty string.